src/data/account.py

```diff
@@ -17,6 +22,34 @@
     from data.event import Event
 
 
+@dataclass
+class Role:
+    stored_role: StoredRole
+
+    @property
+    def role_kind(self) -> RoleKind:
+        return RoleKind(self.stored_role.role)
```

Comment:
```suggestion
        return RoleType(self.stored_role.role)
```
We have `{entity}Type` everywhere, feels out of place to have `Kind` here